pynitrokey/nk3/admin_app.py

```diff
@@ -148,3 +175,24 @@ def set_config(self, key: str, value: str) -> None:
         reply = self._call(AdminCommand.SET_CONFIG, data=request, response_len=1)
         assert reply
         ConfigStatus.check(reply[0], "Failed to set config value")
+
+    def factory_reset(self) -> None:
+        try:
+            reply = self._call(AdminCommand.FACTORY_RESET, response_len=1)
+            assert reply
```

Comment:
That causes a very generic error message if the command is not supported:

```
Critical error:
An unhandled exception occurred
	Exception encountered: AssertionError()
```

Please raise a useful error with `support_hint=False`.